### PR TITLE
Implement command line parsing via clap crate

### PR DESCRIPTION
**Pull request by dburgener**
Also support multiple files on command line

### CODE REVIEW

File: src/bin/casc.rs [hunk 1]
```diff
@@ -3,23 +3,25 @@
...
 fn main() -> std::io::Result<()> {
-    // TODO: Move all this to a parse args function
-    let args: Vec<String> = env::args().collect();
-    if args.len() != 2 {
-        usage();
-        return Err(Error::new(ErrorKind::InvalidInput, "Missing policy file"));
-    }
+    let matches = App::new("casc")
+        .version(VERSION)
+        .author("Daniel Burgener <dburgener@linux.microsoft.com>")
```
Comment (l0kod) [inline src/bin/casc.rs:15]: You can use the CARGO_PKG_AUTHORS env as well.

File: src/bin/casc.rs [hunk 2]
```diff
@@ -3,23 +3,25 @@
...
+            Arg::with_name("INPUT_FILE")
+                .help("Cascade policy files to parse")
+                .required(true)
+                .multiple(true),
+        )
+        .get_matches();
 
-    let policies: Vec<&str> = vec![&args[1]];
+    let policies: Vec<&str> = matches.values_of("INPUT_FILE").unwrap().collect();
     let mut out_file = File::create("out.cil")?;
```
[the remark this answers is not in the document]
Comment (l0kod) [reply src/bin/casc.rs:25]: This would require to update the tools/update-expected-cil.sh BTW.

[the remark this answers is not in the document]
Comment (dburgener) [reply src/bin/casc.rs:25]: I'm planning to add that in a follow-up PR after this one merges.

File: src/bin/casc.rs (hunk 1 above)
Comment (dburgener) [reply src/bin/casc.rs]: Looks like clap offers a crate_authors!() convencience macro to handle the formatting, so I switched to that.  Thanks!

**Comment by l0kod** [comment]
LGTM! Just need to squash the last commits.

**Comment by dburgener** [comment]
I'll do a squash merge if you just want to hit approve.

**Review by l0kod — APPROVED** [review]

File: src/bin/casc.rs (hunk 1 above)
Comment (l0kod) [reply src/bin/casc.rs]: Hmm, there is also `crate_version!()` :)

Comment (dburgener) [reply src/bin/casc.rs]: Good catch!  Let me submit a PR for that.

**Pull request merged**